.vib/pinniped/ginkgo/pinniped_test.go

```diff
@@ -0,0 +1,105 @@
+)
+
+var _ = Describe("Pinniped:", func() {
+	var coreclient cv1.CoreV1Interface
+	var ctx context.Context
+
+	BeforeEach(func() {
+		coreclient = cv1.NewForConfigOrDie(clusterConfigOrDie())
+		ctx = context.Background()
+	})
+
+	When("a testing pod is created", func() {
+		var createdPod *v1.Pod
+
+		BeforeEach(func() {
+			createdPod = createPodOrDie(ctx, coreclient)
+
+			// We need to make sure the pod is out of the "ContainerCreating" phase
+			// to avoid errors when checking its logs
+			isPodRunning, err := retry("isPodRunning", 5, 3*time.Second, func() (bool, error) {
+				return isPodRunning(ctx, coreclient, createdPod.GetName())
+			})
+			if err != nil {
+				panic(err.Error())
+			}
+			Expect(isPodRunning).To(BeTrue())
+		})
+
+		AfterEach(func() {
+			// Not need to panic here if failed, the cluster is expected to clean up with the undeployment
+			coreclient.Pods(*namespace).Delete(ctx, createdPod.GetName(), metav1.DeleteOptions{})
+		})
+
+		Describe("the logs show that", func() {
+			var minidebLogs []string
+
+			BeforeEach(func() {
+				pattern := "Script finished correctly"
+
+				containsPattern, err := retry("containerLogsContainPattern", 5, 2*time.Second, func() (bool, error) {
+					return containerLogsContainPattern(ctx, coreclient, createdPod.GetName(), pattern)
+				})
+				if err != nil {
+					panic(err.Error())
+				}
+				Expect(containsPattern).To(BeTrue())
+
+				minidebPods := getPodsByLabelOrDie(ctx, coreclient, "app=vib-minideb")
+				minidebLogs = getContainerLogsOrDie(ctx, coreclient, minidebPods.Items[0].GetName(), "vib-minideb")
+
+				// Debug code left as is, given configuration complexity
+				if !containsPattern {
+					fmt.Println("##### MINIDEB LOGS: #####")
+					fmt.Println(minidebLogs)
+					fmt.Println("###############")
+
+					openldapPods := getPodsByLabelOrDie(ctx, coreclient, "app=openldap")
+					openldapLogs := getContainerLogsOrDie(ctx, coreclient, openldapPods.Items[0].GetName(), "openldap")
+					fmt.Println("##### OPENLDAP LOGS: #####")
+					fmt.Println(openldapLogs)
+					fmt.Println("###############")
+
+					supervisorPods := getPodsByLabelOrDie(ctx, coreclient, "app.kubernetes.io/component=supervisor")
+					supervisorLogs := getContainerLogsOrDie(ctx, coreclient, supervisorPods.Items[0].GetName(), "supervisor")
+					fmt.Println("##### SUPERVISOR LOGS: #####")
+					fmt.Println(supervisorLogs)
+					fmt.Println("###############")
+				}
+			})
+
+			It("the script was executed entirely", func() {
+				containsPattern, _ := containsPattern(minidebLogs, "Script finished correctly")
+				Expect(containsPattern).To(BeTrue())
+			})
+
+			It("the pinniped cli produced the kubeconfig", func() {
+				containsPattern, _ := containsPattern(minidebLogs, "validated connection to the cluster")
+				Expect(containsPattern).To(BeTrue())
+			})
+
+			It("the pinniped cli configured the cluster auth", func() {
+				containsPattern, _ := containsPattern(minidebLogs, "Username: vibuser")
```

Comment:
I think you should use the `auth-user` variable here instead of the plain `vibuser` value.